authorization/index.md

```diff
@@ -608,13 +613,49 @@ The response is a JSON object containing the access token, with the following ke
     <tr>
       <td><code>scope</code></td>
       <td><span class="label label-success">required</span></td>
-      <td>Scope of access authorized. Note that this can be different from the scopes requested by the app.</td>
+      <td>Scope of access authorized. Note that this will be the same as the scope of the original access token, and it can be different from the scopes requested by the app.</td>
     </tr>
     <tr>
       <td><code>refresh_token</code></td>
       <td><span class="label label-info">optional</span></td>
       <td>The refresh token issued by the authorization server. If present, the app should discard any previosu <code>refresh_token</code> associated with this launch, replacing it with this new value.</td>
     </tr>
+    <tr>
+      <td><code>host</code></td>
+      <td><span class="label label-success">required</span></td>
+      <td>URL of the server that issued the token.</td>
+    </tr>
+    <tr>
```

Comment:
Please see my comments on #31 , which apply here, too. I think we've got our wires crossed on this one -- I'm not sure what `username`, `timestamp`, and `client_id` are all about (please let's continue to discuss in #31 -- I'm waiting to hear back from you).

I also don't think the original launch-specific context (`intent` or styling) necessarily apply here.